docs/io/unified.rst

```diff
@@ -5,7 +5,7 @@ Unified File Read/Write Interface
 
 ``astropy`` provides a unified interface for reading and writing data in
 different formats. For many common cases this will streamline the process of
-file I/O and reduce the need to master the separate details of all of the I/O
+file I/O and reduce the need to learn the separate details of all of the I/O
```

Comment:
You know, I'm not really sure this change is *necessary* since I think this is the one use of "master" that has a different etymology.  But  "learn" actually works fine here and sounds better to me anyway.  Just not something we necessarily have to stay vigilant about in the future :shrug:.

Reply:
It was a blind `sed` run and I thought "might as well"...